src/popup/components/AddressLink.vue

```diff
@@ -0,0 +1,76 @@
+<template>
```

Comment:
There is already a component that looks the same: `<AccountItem />`. Only difference is that its not clickable. But this can be easily tweaked.